lab-phelan/test/greet.test.js

```diff
@@ -0,0 +1,9 @@
+const greet = require(`../lib/greet`);
+
+test('Testing greet > sayHi function.', () => {
+  expect(greet('world')).toBe('hello world');
```

Comment:
you don't always have to use toBe, you can also use toEqual, toBeTruthy, toBeFalsy, toMatch, toBeGreaterThan